modules/github_integration/frontend/module/tab-prs/tab-prs.component.spec.ts

```diff
@@ -109,11 +149,11 @@ describe('TabPrsComponent', () => {
     fixture = TestBed.createComponent(TabPrsComponent);
     component = fixture.componentInstance;
     element = fixture.debugElement;
-    halResourceService = fixture.debugElement.injector.get(HalResourceService) as jasmine.SpyObj<HalResourceService>;
     changeDetectorRef = fixture.debugElement.injector.get(ChangeDetectorRef) as jasmine.SpyObj<ChangeDetectorRef>;
     // @ts-ignore
     component.workPackage = { id: 'testId' };
 
+    changeDetectorRef.markForCheck();
```

Comment:
manually invoking change detection gets this spec passing - not sure if this is the canonical way of testing asynchronous in Angular tests though?